From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Tue, 9 Mar 2021 14:46:27 +0100
Subject: location-panel: Allow label to wrap

Otherwise the panel overflows when enabling location services.

Upstream-Reference: https://gitlab.gnome.org/GNOME/gnome-control-center/-/merge_requests/964
---
 panels/location/cc-location-panel.ui | 1 +
 1 file changed, 1 insertion(+)

diff --git a/panels/location/cc-location-panel.ui b/panels/location/cc-location-panel.ui
index 9bceb9c..c90de44 100644
--- a/panels/location/cc-location-panel.ui
+++ b/panels/location/cc-location-panel.ui
@@ -90,6 +90,7 @@
                         <property name="visible">true</property>
                         <property name="halign">start</property>
                         <property name="xalign">0</property>
+                        <property name="wrap">1</property>
                         <property name="label" translatable="yes">Allow the applications below to determine your location.</property>
                       </object>
                       <packing>
